data/mockable.rb

```diff
@@ -0,0 +1,51 @@
+require 'Date'
+
+module Mockable
+  extend self
+
+  def created_at_proc
+    return Proc.new { |date| next date.next_year }
+  end
```

Comment:
Cool idea to put this into a Module! What's the different with using `extend` again vs using `include`?

Reply:
extend causes the methods in the module to bind at the class level. Include binds are the instance level